app/queries/proxies_for_owner_and_watcher_query.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class ProxiesForOwnerAndWatcherQuery
+  def initialize(owner:, watcher: nil)
```

Comment:
The words "owner" and "watcher" are a bit counter-intuitive considering how we name other things in general in Porta. It's like introducing new concepts for things we already know by different names. "owner" looks more like "scope" to me, while "watcher" is a "user".

Reply:
changing names is ok for me, i can agree that it can be better named. However, I disagree with the names that you suggested:
- 'user' sounds like an instance of a User, and here it means 'whoever is logged in, whether is a **user** or an **account**'
- 'scope' is definitely not even similar to 'owner'. 'owner' is like we call a similar concept in attributes of AccessToken, Metric, DeletedObject, ProxyRule, etc.